src/patternfly/components/AboutModalBox/examples/about-modal-box-example.hbs

```diff
@@ -4,7 +4,8 @@
     {{/about-modal-box-brand-image}}
   {{/about-modal-box-brand}}
   {{#> about-modal-box-close about-modal-box-close--attribute='aria-label="Close Dialog"'}}
-    {{#> button button--modifier="pf-m-plain" button--HasTimesIcon="true"}} 
+    {{#> button button--modifier="pf-m-plain"}}
+      {{> button-icon button-icon--type="times" button-icon--modifier="pf-c-button__icon"}} 
```

Comment:
Looks like this button is off-center now. Can you track that down and address it, and check if there are any other instances of this?
<img width="107" alt="screen shot 2019-01-04 at 9 15 17 am" src="https://user-images.githubusercontent.com/35148959/50695084-46c77800-1001-11e9-9d61-9e43e095851d.png">